cloud_infrastructure_access/dedicated-understanding-aws.adoc

```diff
@@ -0,0 +1,25 @@
+:context: dedicated-understanding-aws
+[id="welcome-index"]
+= Understanding Cloud Infrastructure Access
+include::modules/common-attributes.adoc[]
+
+Amazon Web Services (AWS) infrastructure access allows
+link:https://access.redhat.com/node/3610411[Customer Portal Organization Administrators]
+and cluster owners to enable AWS Identity and Access Management (IAM) users to
+have federated access to the AWS Management Console for their {product-title}
+cluster.
+
+== Enabling AWS Access
+AWS access can be granted for customer AWS users, and private cluster access can be implemented to suit the needs of your {product-title} environment.
+
+Get started with xref:../cloud_infrastructure_access/dedicated-aws-access.adoc#dedicated-aws-access[Accessing AWS Infrastructure] for your {product-title} cluster. This will guide you through creating an AWS user and account and providing that user with access to the {product-title} AWS account.
```

Comment:
s/Infrastructure/infrastructure (here and in the other assembly)

I'd s/. This will guide you through/by